src/components/Todo/TodoContainer.tsx

```diff
@@ -10,11 +10,7 @@ export const TodoContainer: React.FC<TodoContainerProps> = ({
   description,
   deleteTodo,
 }) => {
-  const [avaliableOptions, setAvaliableOptions] = useState([
-    'Todo',
-    'In progress',
-    'Done',
-  ]);
+  const availiableOptions = ['Todo', 'In progress', 'Done'];
```

Comment:
Store this array out of the component. It'll help with memoization.

Reply:
Should i store it outside the func body?